manage_people/templates/manage_people/results_list.html

```diff
@@ -3,7 +3,15 @@
 {% load manage_people_tags %}
 {% load static %}
 
-{% block breadcrumbs %}<a href="{% url 'manage_people:user_form' %}">Manage People</a> <i class="fa fa-chevron-right"></i> Add People{% endblock breadcrumbs %}
+{% block breadcrumbs %}
+<nav>
+    <h3>
+        <a href="{% url 'dashboard_course' %}">Manage Course</a>&nbsp;<i class="fa fa-chevron-right"></i>&nbsp;
+        <a href="{% url 'manage_people:user_form' %}">Manage People</a>&nbsp;<i class="fa fa-chevron-right"></i>&nbsp;
+        <a href="{% url 'manage_people:find_user' %}">Search for people</a>&nbsp;<i class="fa fa-chevron-right"></i>&nbsp;Add People
```

Comment:
The 'Search for people' link is not in the original tool -- did the ACs change? I must be missing something.